src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+
```

Comment:
please recheck all models and remove first empty line for all occurrences
they are not required